### DISCUSSION TITLE
Document response headers

### DISCUSSION THREAD

**Discussion by chbndrhnns** [fastapi/fastapi#4849] [Questions] [answered]
Is there a programmatic way to document response headers in the API spec using FastAPI?

**Comment by jgould22** [comment, marked as answer]
You may find this part of the docs useful 

https://fastapi.tiangolo.com/advanced/extending-openapi/#modify-the-openapi-schema

and 

https://swagger.io/docs/specification/describing-responses/

**Comment by chbndrhnns** [comment]
Thanks much! I guess that's as programmatic as it gets with the current implementation. I assume it's added to the spec directly and no validation is in place which would reject data not conforming to that additional metadata.
